docs/contributions.rst

```diff
@@ -360,6 +361,8 @@ Committing and submitting a pull request
 * Wait for 7 days at least and get an LTGM comment from a member of the
   team if your commit changes the other parts than your parser.
 
+.. TODO: should be LGTM (Looks Good To Me) ?
```

Comment:
LGTM is needed (1) when the change you made is for files other than you maintain and (2) when the change is not obivous.

Fixing a typo is so obvious, so you don't have to wait LGTM response.

Reply:
Fixed the typo and add description.